contracts/modules/RelayerManager.sol

```diff
@@ -92,6 +92,8 @@ abstract contract RelayerManager is BaseModule, SimpleOracle {
         external
         returns (bool)
     {
+        require(!_isLocked(_wallet) || _gasPrice == 0, "RM: Locked wallet refund");
```

Comment:
We should move this check a few lines down (e.g. after `require(verifyData(_wallet, _data), "RM: Target of _data != _wallet");` so that it is included in the gas refund accounting.